seqr/management/tests/add_project_tag_tests.py

```diff
@@ -45,20 +45,20 @@ def test_missing_required_args(self):
         out = StringIO()
         with self.assertRaises(CommandError) as err:
             call_command('add_project_tag', stdout = out)
-        self.assertEqual(err.exception.message, 'Error: argument --project is required')
+        self.assertRegex(str(err.exception), 'Error:.*argument.*--project')
```

Comment:
In order to make sure the error message is as expected, could we change this to check that the error is one of the 2 exact messages we expect, rather than allowing anything with the argument name:
```
self.assertTrue(str(err.exception) in {'Error: argument --project is required', 'Error: the following arguments are required: --project'})
```

Reply:
Okey, I will do that.

Reply:
How about use `assertIn()` as below?
```
self.assertIn(str(err.exception), ['Error: argument --project is required',
       'Error: the following arguments are required: --project, --name, --order'])
```